src/main/webapp/app/pages/newsPage/Util.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { ElementType } from 'app/components/SimpleTable';
+import { GenePageLink } from 'app/shared/utils/UrlUtils';
+import WithSeparator from 'react-with-separator';
+
+export const convertGeneInputToLinks = (geneInput: string): ElementType => {
+  geneInput = geneInput.trim();
+  const tokens = geneInput.split(',');
+  if (tokens.length === 1) {
+    return <GenePageLink hugoSymbol={geneInput} />;
+  }
+
+  const itemLinks = tokens.map(token => {
+    token = token.trim();
+    return <GenePageLink hugoSymbol={token} />;
+  });
+
+  return <WithSeparator separator=", ">{itemLinks}</WithSeparator>;
+};
+
+export const getGeneColumnIndex = (
+  annotationColumnHeader: {
+    name: string;
+  }[]
+): number => {
+  return annotationColumnHeader.findIndex(column => column.name === 'Gene');
```

Comment:
I think I didn't make it clear last time. You could make `Gene` a constant so it can be used here and also for all `CHANGED_ANNOTATION_ADDITIONAL_*`definitions.